app/config/services/services.xml

```diff
@@ -596,6 +595,12 @@
             <argument type="string">%env(MAILCHIMP_MEMBER_INTEREST_INTEREST_GROUP_ID)%</argument>
             <argument type="string">%env(MAILCHIMP_SUBSCRIPTION_TYPE_INTEREST_GROUP_ID)%</argument>
         </service>
+        <service id="AppBundle\Mailchimp\Campaign\Listener\SetCampaignReplyToListener">
+            <tag name="kernel.event_subscriber"/>
```

Comment:
Why adding manually this tag ? the `autoconfigure` is enabled.

Reply:
oh yes, fixed, thanks